database/src/main/resources/db/migration/V1_133__detected_route_changes_table.sql

```diff
@@ -0,0 +1,91 @@
+CREATE TABLE "detected-route-change" (
+ "transit-change-date"        DATE NOT NULL,
+ "transit-service-id"         INTEGER NOT NULL references "transport-service" (id),
+ "route-short-name"           TEXT,                   -- e.g. "15"
+ "route-long-name"            TEXT NOT NULL,          -- e.g. "Helsinki-Oulu"
+ "trip-headsign"              TEXT,
+ "change-type"                "gtfs-route-change-type", 
+ "added-trips"                INTEGER DEFAULT 0,
+ "removed-trips"              INTEGER DEFAULT 0,
+ "trip-stop-sequence-changes-lower"   INTEGER DEFAULT 0 NOT NULL, -- Range of change counts in trips (0 - 2), low limit
+ "trip-stop-sequence-changes-upper"   INTEGER DEFAULT 0 NOT NULL, -- Range of change counts in trips (0 - 2), high limit
+ "trip-stop-time-changes-lower"       INTEGER DEFAULT 0 NOT NULL, -- Range of stop time changes in trips (0 - 200), low limit
+ "trip-stop-time-changes-upper"       INTEGER DEFAULT 0 NOT NULL, -- Range of stop time changes in trips (0 - 200), high limit
+ "current-week-date"          DATE,                   -- Day of detection run
```

Comment:
Just making notes: We should change this column name when we change that detection algorithm.